packages/grafana-ui/src/components/Select/types.ts

```diff
@@ -103,6 +105,14 @@ export interface SelectAsyncProps<T> {
   loadingMessage?: string;
 }
 
+/** The VirtualizedSelect component uses a slightly different SelectableValue, description and other props are not supported */
+export interface VirtualizedSelectProps<T> extends Omit<SelectCommonProps<T>, 'virtualized'> {
+  options?: Array<{
+    value: T;
+    label: string;
+  }>;
+}
```

Comment:
```suggestion
export interface VirtualizedSelectProps<T> extends Omit<SelectCommonProps<T>, 'virtualized'> {
  options?: Array<{
    value?: T;
    label?: string;
  }>;
}
```

Reply:
Or maybe even something like this:

```suggestion
export interface VirtualizedSelectProps<T> extends Omit<SelectCommonProps<T>, 'virtualized'> {
  options?: Array<Pick<SelectableValue<T>, 'label' | 'value'>>;
}
```

Reply:
Absolutely, great suggestion! 🙌